utils/drl_algorithm_selector.py

```diff
@@ -0,0 +1,25 @@
+from stable_baselines3 import PPO, DQN, A2C
+
+class ModelSelector:
+    def __init__(self, model_type, env, config):
+        self.model_type = model_type
+        self.env = env
+        self.config = config
+        self.model = self.init_model()
+
+    def init_model(self):
```

Comment:
Consider refactoring this method to be more modular by using helper methods or a dictionary to map `model_type` to configurations and constructors.